### DISCUSSION TITLE
SDHC vs. SDXC

### DISCUSSION THREAD

**Discussion by kuyachris91** [gtxaspec/wz_mini_hacks#291] [Q&A]
Hi all,

I'm working with a Wyze Cam v3 on firmware 4.36.9.139. I was able to get wz_mini_hacks up and running using a 16GB microSD SDHC card. When I tried using my other SD cards that were SDXC (64GB/128GB) it didn't work.

Is this a known issue? I also made sure that the SDXC cards were formatted to FAT32 as per the instructions — they were NOT formatted as exFAT. I reformatted using both the MacOS Disk Utility and terminal `sudo diskutil eraseDisk FAT32`

Just making this note since I didn't see specific notes between SDHC and SDXC microSD cards. I'm wondering if it's just an error on my end. Has other folks gotten wz_mini_hacks running using higher capacity SDXC cards?

**Comment by clee604** [comment]
I use wz_mini_hacks with 256gb sandisk and samsung cards with no issues. I don't use a Mac though, I formatted the cards using Linux or ChromeOS and both methods seem to work fine for me

**Comment by kuyachris91** [comment]
Ah, I might have found the difference. When I format on Mac, using both Disk Utility or terminal, the FAT32 format type is `DOS_FAT_32` when I run `diskutil list` while the SD card that worked is formatted as `Windows_FAT_32`. So I think it's the specific FAT32 format that's an issue, not SDHC vs. SDXC.

**Comment by kuyachris91** [comment]
Alright, just as an update for reference. I had another Wyze cam with older firmware and the install works even if the SDXC card is formatted as `DOS_FAT_32` — working on a Wyze V3 using a 64GB and 128GB SDXC card on firmware version 4.36.3.19.

I tried the exact same SD card and config files on two Wyze V3 cameras and the only real difference I can see is the firmware version:

- 4.36.9.139
  - 16GB micoSD SDHC - working
  - 64GB microSD SDXC - not working
  - 128GB microSD SDXC - not working
- 4.36.3.19
  - 16GB micoSD SDHC - working
  - 64GB microSD SDXC - working
  - 128GB microSD SDXC - working

I test the install by remotely connecting though ssh and viewing the RTSP stream. Not really sure what's going on.

**Reply by endertable** [reply]
Wyze just got a license for exFat and if SD card has dirty bit set, it *might* try reformatting the card to exFat as that is their preferred method for cards over 32GB in size (truly, anything IS better than FAT or FAT32 as the dirty bit gets set very easily on these).  That update only pertains to their latest software so not sure if that might be an issue that is happening with your larger cards on the newer FW.

**Comment by virmaior** [comment]
this is basically: https://github.com/gtxaspec/wz_mini_hacks/discussions/90
repeated.

Are you formatting with:
```
diskutil partitionDisk /dev/disk4 1 MBRFormat "MS-DOS FAT32" NONAME R

```
?
